sql/CMakeLists.txt

```diff
@@ -1123,294 +1123,7 @@ ENDIF()
 OPTION(DEBUG_EXTNAME "Build server as mysqld-debug (debug builds only)" OFF)
 MARK_AS_ADVANCED(DEBUG_EXTNAME)
 
-INCLUDE(${CMAKE_SOURCE_DIR}/cmake/bison.cmake)
```

Comment:
Can you check if we need to add an additional condition to avoid including bison in main CMakeLists.txt

> IF(NOT WITHOUT_SERVER)
  INCLUDE(bison)
ENDIF()

Reply:
@rahulmalik87 we only add bison in sql/CMakeLists.txt if we are building server:
https://github.com/altmannmarcelo/percona-xtrabackup/blob/PXB-2966-8.0/CMakeLists.txt#L1923